xml/cap_admin_minibroker.xml

```diff
@@ -0,0 +1,291 @@
+    <itemizedlist>
+     <listitem>
+      <para>
+       <link xlink:href="https://mariadb.org">MariaDB</link>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <link xlink:href="https://www.mongodb.com/">MongoDB</link>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <link xlink:href="https://www.postgresql.org/">PostgreSQL</link>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <link xlink:href="http://redis.io">Redis</link>
+      </para>
+     </listitem>
+    </itemizedlist>
+    <warning>
+     <title>Do not use Minibroker services on production systems</title>
+     <para>
+      The services used by Minibroker are not production grade and are not recommended for use with production systems.
+     </para>
+    </warning>
+   </step>
+   <step>
+    <para>
+     Monitor the deployment progress. Wait until all pods are in a ready state
+     before proceeding:
+    </para>
+<screen>&prompt.user;watch -c 'kubectl get pods --namespace minibroker'</screen>
+   </step>
+  </procedure>
+ </sect1>
+ <sect1 xml:id="sec.cap.minibroker-environment-setup">
+  <title>Setting up the environment for Minibroker usage</title>
+
+  <procedure>
+   <step>
+    <para>
+     Begin by logging into your &cap; deployment. Select an organization and
+     space to work with, creating them if needed:
+    </para>
+<screen>&prompt.user;cf api --skip-ssl-validation <replaceable>https://api.example.com</replaceable>
+&prompt.user;cf login -u <replaceable>admin</replaceable> -p <replaceable>password</replaceable>
+&prompt.user;cf create-org <replaceable>org</replaceable>
+&prompt.user;cf create-space <replaceable>space</replaceable>
+&prompt.user;cf target -o <replaceable>org</replaceable> -s <replaceable>space</replaceable>
+</screen>
+   </step>
+   <step>
+    <para>
+     Create the service broker. Note that Minibroker does not require
+     authentication and the <literal>username</literal> and
+     <literal>password</literal> parameters act as dummy values to pass to the
+     <command>cf</command> command. These parameters do not need to be
+     customized for the &cap; installation:
+    </para>
+<screen>&prompt.user;cf create-service-broker <replaceable>minibroker</replaceable> <replaceable>username</replaceable> <replaceable>password</replaceable> http://minibroker-minibroker.minibroker.svc.cluster.local</screen>
+    <para>
+     Once the service broker is ready, it can be seen on your deployment:
+    </para>
+<screen>&prompt.user;cf service-brokers
+Getting service brokers as admin...
+
+name               url
+minibroker         http://minibroker-minibroker.minibroker.svc.cluster.local
+</screen>
+   </step>
+   <step>
+    <para>
+     Enable access to a service. Services that can be enabled are
+     <literal>mariadb</literal>, <literal>mongodb</literal>,
+     <literal>postgresql</literal>, and <literal>redis</literal>. The example
+     below will uses Redis as the service:
```

Comment:
Delete "will"

Reply:
Fixed in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68